pmd-apex/src/test/resources/net/sourceforge/pmd/lang/apex/rule/errorprone/xml/AvoidDirectAccessTriggerMap.xml

```diff
@@ -20,9 +20,9 @@ trigger AccountTrigger on Account (before insert, before update) {
         <expected-problems>0</expected-problems>
         <code><![CDATA[
 trigger AccountTrigger on Account (before insert, before update) {
-   foreach ( Account a : Trigger.new ){   
+   for ( Account a : Trigger.new ){
```

Comment:
We also should fix the example: https://pmd.github.io/latest/pmd_rules_apex_errorprone.html#avoiddirectaccesstriggermap :smile: